reconcile/test/test_gabi_authorized_users.py

```diff
@@ -92,9 +97,10 @@ def test_gabi_authorized_users_apply(
         secret_read,
         get_settings,
     ):
-        expirationDate = date.today()
+        expirationDate = date(2023, 1, 1)
```

Comment:
Please rename the variable to snake_case `expiration_date`. 🙇

Reply:
Done in https://github.com/app-sre/qontract-reconcile/pull/3219/commits/4dfd15cba52e4610bce6f43c2b535e3d5c2f48b1. Thank you!